### PR TITLE
Step towards removing RenderSimple and its derivatives (Part 1)

### PR DESCRIPTION
**Pull request by reaperrr**
This PR gets some of the easy stuff out of the way to reduce the diff and improve reviewability of the follow-ups.

### CODE REVIEW

**Comment by GraionDilach** [comment]
I'd say the YAML changes of the mines should be extracted out to a new PR with removing the RA1 mod defaults.

File: OpenRA.Mods.Common/Traits/Render/WithSpriteBody.cs [hunk 1]
```diff
@@ -81,7 +81,7 @@ public void PlayCustomAnimation(Actor self, string name, Action after = null)
 
 		public void PlayCustomAnimationRepeating(Actor self, string name)
 		{
-			DefaultAnimation.PlayThen(name,
+			DefaultAnimation.PlayThen(NormalizeSequence(self, name),
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/Render/WithSpriteBody.cs]: You normalize this one, but not the one below?

**Comment by reaperrr** [comment]
Updated.

**Comment by Mailaender** [comment]
Looks sensible. :+1: Editor only rendering still works. :white_check_mark:

File: OpenRA.Mods.Common/Traits/Render/WithSpriteBody.cs [hunk 2]
```diff
@@ -81,8 +81,8 @@ public void PlayCustomAnimation(Actor self, string name, Action after = null)
 
 		public void PlayCustomAnimationRepeating(Actor self, string name)
 		{
-			DefaultAnimation.PlayThen(name,
-				() => PlayCustomAnimationRepeating(self, name));
+			DefaultAnimation.PlayThen(NormalizeSequence(self, name),
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/Render/WithSpriteBody.cs]: Harvesting animations now appear to be double-normalized.  You should be able to adjust WithHarvestAnimation to account for these upstream checks (which I think are a good idea).

**Comment by reaperrr** [comment]
Updated.
For the record, I did test the harvester normalization in combination with damage stages, seems to work as intended.

**Comment by reaperrr** [comment]
Updated, removed the `RenderSimple` commit. It will later return in its own PR, where `RenderSimple` will actually be removed from code as well. That PR will follow the complete removal of `RenderBuilding`.

**Comment by reaperrr** [comment]
Updated.

**Comment by penev92** [comment]
Needs a rebase.

**Comment by penev92** [comment]
Looks and works OK.
:+1: when my comment on the last commit is addressed. Also when rebased, of course.

**Comment by reaperrr** [comment]
Updated and rebased.

**Comment by reaperrr** [comment]
Updated.

**Comment by penev92** [comment]
Thanks.
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/f307594884e7c8cd2552a8e857bade75ae05d0e6...c9903db9f8a964dc11006a3306022f18e3827045)

**Pull request merged**